e2e/components/NotificationsPanel/NotificationsPanel-test.avt.e2e.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright IBM Corp. 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+import { expect, test } from '@playwright/test';
+import { visitStory } from '../../test-utils/storybook';
+
+test.describe('NotificationsPanel @avt', () => {
+  test('@avt-default-state', async ({ page }) => {
+    await visitStory(page, {
+      component: 'NotificationsPanel',
+      id: 'ibm-products-patterns-notifications-notificationspanel--default',
+      globals: {
+        carbonTheme: 'white',
+      },
+    });
+
+    const notificationPanelElement = page.locator(
+      `#dev-prefix--c4p--notifications-panel`
```

Comment:
You should be able to pull in our prefix here.

```jsx
import { pkg } from '../../../ibm-products/src/settings';
...
...
...
const notificationPanelElement = page.locator(
    `#${pkg.prefix}--notifications-panel`
);
```

Reply:
had to do 
```
import { pkg } from '../../../packages/ibm-products/src/settings';

...

 const notificationPanelElement = page.locator(
      `#dev-prefix--${pkg.prefix}--notifications-panel`
    );
```

since it looks like we update the prefix in storybook to `dev-prefix--c4p`